style.css

```diff
@@ -108,3 +113,166 @@ h1,
   display: flex;
   justify-content: center;
 }
+
+/* recent-works  /////////////////////// */
+
+.recent-works {
+  padding: 24px;
+  text-align: center;
+}
+
+.text {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  padding-bottom: 40px;
+}
+
+.card-work-row {
+  display: grid;
+  row-gap: 24px;
+  padding: 20px 0;
+}
+
+.card {
+  height: 475px;
+  background-color: #ebf0ee;
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.text-box {
+  height: 50%;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 8px;
+}
+
+.project-title {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+}
+
+.tags {
+  display: flex;
+  column-gap: 8px;
+}
+
+.tag {
+  background-color: #ebf0ee;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+  padding: 8px 12px;
+  border-radius: 4px;
+}
+
+.see-project {
+  background-color: #36b37f;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: white;
+  padding: 12px;
+  border-radius: 4px;
+}
+
+.see-project:hover {
+  color: #ebf0ee;
+}
+
+.see-project:active {
+  color: #36b37e;
+}
+
+/* about me /////////////////////////// */
+
+.about-me {
+  padding: 60px 12px;
+}
+
+.about-text {
+  background-image: url(images/backgrounds/top.svg), url(images/backgrounds/bottom.svg);
+  background-repeat: no-repeat;
+  background-position: right top, left bottom;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: 22px;
+  padding-bottom: 50px;
+}
+
+.primary-text {
+  font-weight: 700;
+  line-height: 52px;
+  font-size: 40px;
+  color: #172b4d;
+}
+
+.supporting-text-1 {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #42526e;
+}
+
+/* .see-project{ } is actually done previously */
+
+.about-cards {
+  display: grid;
+  row-gap: 24px;
+}
+
+.card-works {
+  height: 350px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.img-icon {
+  background-color: white;
+  padding: 20px;
+  border-radius: 50%;
+}
+
+.languages-title {
+  margin-top: -55px;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #344563;
+}
+
+.about-tags {
+  margin-top: -90px;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  column-gap: 12px;
+  row-gap: 12px;
+}
+
```

Comment:
- [ ] Please 🙏 add a bit more margin between the skill's title and stack to make it look like[ Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=D7dAJHCksl2FeGQL-0) as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) of the project to stick to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=D7dAJHCksl2FeGQL-0)

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

_**Figma Design**_

<img width="151" alt="Screenshot 2023-02-18 at 10 52 43 AM" src="https://user-images.githubusercontent.com/44798044/219843973-ae1a8e86-515a-444c-9c4e-51fdac223e3b.png">

**_Your Design_**

<img width="321" alt="Screenshot 2023-02-18 at 10 52 55 AM" src="https://user-images.githubusercontent.com/44798044/219843988-dff7c43e-439d-4edf-82c6-008a639a8065.png">